packages/@react-spectrum/textfield/src/TextArea.tsx

```diff
@@ -25,22 +25,33 @@ function TextArea(props: SpectrumTextFieldProps, ref: RefObject<TextFieldRef>) {
     isReadOnly = false,
     isRequired = false,
     onChange,
+    defaultValue,
+    value,
     ...otherProps
   } = props;
 
   let inputRef = useRef<HTMLInputElement & HTMLTextAreaElement>();
+  let [count, check] = useState(0);
 
-  let onHeightChange = () => {
-    if (isQuiet) {
-      let input = inputRef.current;
-      input.style.height = 'auto';
-      input.style.height = `${input.scrollHeight}px`;
+  function changeDomHeight(dom: HTMLInputElement & HTMLTextAreaElement) {
+    dom.style.height = 'auto';
+    dom.style.height = `${dom.scrollHeight}px`;
+  }
+
+  let onCheckHeight = () => check(x => x + 1);
+
+  useEffect(() => {
+    if (isQuiet && inputRef.current) {
+      if (!defaultValue && !value && count === 0) {
```

Comment:
what is this check for? can there be a comment explaining it?

Reply:
If we call `onHeightChange` on component mount, the test for `quiet variant automatically adjusts its vertical height on change`, which no `defaultValue` prop or `value` prop is passed to `TextArea` component, will be failed because the test logic is like, if no `defaultValue` prop or `value` prop is passed to `TextArea` then the DOM's height should be `''` after mounting .See [TextArea.test.js#L53](https://github.com/adobe/react-spectrum/blob/8560ae2a0a10fdd1e86b220282b3e0c06f795579/packages/%40react-spectrum/textfield/test/TextArea.test.js#L53).

So the check is for that: don't call `onHeightChange` on that situation. I'm not sure if I get this right 🤣 🤣